### PR TITLE
xvisor build Trustcore module.

### PR DESCRIPTION
**Pull request by rdevarajantct**
This needs the TCDIR to be set to the correct path of the module

### CODE REVIEW

File: openconf.cfg [hunk 1]
```diff
@@ -31,4 +31,4 @@ source libs/openconf.cfg
 source drivers/openconf.cfg
 source emulators/openconf.cfg
 source tools/openconf.cfg
-
+source ../poc/tcmod/openconf.cfg
```
Comment (rneiman) [inline openconf.cfg]: should this path be relative to TCDIR?

Comment (rdevarajantct) [reply openconf.cfg]: Unfortunately this one does not take any environment variables and hence it has to be hard coded. Can't find any other alternative for this.

Comment (rneiman) [reply openconf.cfg]: does this mean that we need to take care of providing this file at this exact path?

Comment (rdevarajantct) [reply openconf.cfg]: Yes, we need to adjust this path based on the location of the file.

Comment (rneiman) [reply openconf.cfg]: how about we create this path with its .cfg from the Makefile to avoid manual creation and guarantee correctness?

Comment (rdevarajantct) [reply openconf.cfg]: That would work. Will try and update this PR

Comment (rdevarajantct) [reply openconf.cfg]: updated the PR

File: openconf.cfg [hunk 2]
```diff
@@ -31,4 +31,4 @@ source libs/openconf.cfg
 source drivers/openconf.cfg
 source emulators/openconf.cfg
 source tools/openconf.cfg
-source ../poc/tcmod/openconf.cfg
+source #TCDIR/tcmod/openconf.cfg
```
Comment (rneiman) [inline openconf.cfg:34]: not $(TCDIR)?

Comment (rdevarajantct) [reply openconf.cfg:34]: nope that does not work. sed is not able to change the $TCDIR name, rather it tries to substitute it.

**Review by rneiman — APPROVED** [review]

**Pull request merged**